docs/excel/excel-add-ins-comments.md

```diff
@@ -1,14 +1,11 @@
 ---
-title: Work with comments using the Excel JavaScript API (preview)
+title: Work with comments using the Excel JavaScript API
 description: ''
-ms.date: 10/16/2019
+ms.date: 10/22/2019
```

Comment:
There's a broken link in this article -- the "mentions" bookmark link in the **Add comments** section is broken, because the section has been renamed. Need to update the bookmark target to match the new section name.